ctapipe_io_lst/tests/test_lsteventsource.py

```diff
@@ -266,3 +267,24 @@ def test_pedestal_events(tmp_path):
                 assert event.trigger.event_type == EventType.SKY_PEDESTAL
             else:
                 assert event.trigger.event_type != EventType.SKY_PEDESTAL
+
+
+
+
+@pytest.mark.parametrize(
+    "trigger_bits,expected_type",
+    [
+        (TriggerBits.MONO, EventType.SUBARRAY),
+        (TriggerBits.STEREO, EventType.SUBARRAY),
+        (TriggerBits.CALIBRATION, EventType.FLATFIELD),
+        (TriggerBits.CALIBRATION | TriggerBits.PEDESTAL, EventType.UNKNOWN),
+        (TriggerBits.CALIBRATION | TriggerBits.MONO, EventType.UNKNOWN),
+        (TriggerBits.PEDESTAL, EventType.SKY_PEDESTAL),
+    ]
```

Comment:
@FrancaCassol @moralejo Can you check if these are the combinations / expected results we want to have?